pylint/pyreverse/inspector.py

```diff
@@ -196,8 +184,17 @@ def visit_classdef(self, node: nodes.ClassDef) -> None:
                     self.handle_assignattr_type(assignattr, node)
         # resolve implemented interface
         try:
-            ifaces = interfaces(node, self.inherited_interfaces)
-            node.implements = list(ifaces) if ifaces is not None else []
+            ifaces = interfaces(node)
+            if ifaces is not None:
+                node.implements = list(ifaces)
+                warnings.warn(
+                    "pyreverse will drop support for resolving and displaying implemented interfaces in pylint 3.0. "
+                    "The implementation relies on the '__implements__'  attribute proposed in PEP 245, which was rejected "
+                    "in 2006.",
```

Comment:
Wow, 2006 😄 I was copy pasting javascript and basic for high school calculator at the time.